tests/xml_parser_test.c

```diff
@@ -165,6 +165,40 @@ static int s_xml_parser_siblings_with_text_test(struct aws_allocator *allocator,
 
 AWS_TEST_CASE(xml_parser_siblings_with_text, s_xml_parser_siblings_with_text_test)
 
+const char *siblings_with_empty_tag = "<?xml version=\"1.0\" "
+                                      "encoding=\"UTF-8\"?><rootNode><child1 /><child2>TestBody2</child2></rootNode>";
+
+static int s_xml_parser_child_empty_tag(struct aws_allocator *allocator, void *ctx) {
+    (void)ctx;
+    struct sibling_text_capture capture;
+    AWS_ZERO_STRUCT(capture);
```

Comment:
trivial: instead of zero out the `capture.capture1`, set it to some garbage so that we can test that the parse sets the outcome to empty cursor correctly.

Reply:
ah, good point